### PR TITLE
doc fixup

### PR DESCRIPTION
**Pull request by rlucas7**
tiny fixup to doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-To run training and eval, from the repo root (need at least 40GB mem, 16 cores on arch x86_64, and preferrably a V100 GPU):
+To run training and eval, from the repo root (need at least 40GB mem, 16 cores on arch x86_64, and a GPU for triton dependency)
```
Comment (D-Roberts) [inline README.md:1]: triton dependency was removed so I'd remove the comment about triton.

**Review by D-Roberts — APPROVED** [review]
Minor edit before LGTM!

File: README.md (hunk 1 above)
Comment (rlucas7) [inline README.md:1]:
```suggestion
To run training and eval, from the repo root (need at least 40GB mem, 16 cores on arch x86_64, and preferably a V100 GPU):
```

**Review by rlucas7 — COMMENTED** [review]
use suggestion to edit

**Pull request merged**